src/wallets/browser/NearWallet.ts

```diff
@@ -56,9 +53,7 @@ class NearWallet extends BrowserWallet implements INearWallet {
   }
 
   async signIn() {
-    const state = getState();
-
-    this.wallet.requestSignIn(state.options.accountId).then(() => {
+    this.wallet.requestSignIn(this.options.accountId).then(() => {
```

Comment:
The `requestSignIn` method fails probably we need to call `this.init()` somwhere before trying to access properties of `this.wallet`